### PR TITLE
Implement remove role

### PR DESCRIPTION
**Pull request by mfaizanrasool**
Implement remove role.

### CODE REVIEW

File: inc/class-wpfep-roles-editor.php [hunk 1]
```diff
@@ -442,461 +471,686 @@ public function edit_publish_meta_box($post)
...
+
+	}
+
+	function wp_default_scripts() {
+
+		$wp_default_scripts = array(
+			'jquery',
+			'jquery-core',
+			'jquery-migrate',
+			'jquery-ui-core',
```
Comment (zain-glowlogix) [inline inc/class-wpfep-roles-editor.php]: @mfaizanrasool please remove all unnecessary scripts.

File: inc/class-wpfep-roles-editor.php [hunk 2]
```diff
@@ -442,461 +471,686 @@ public function edit_publish_meta_box($post)
...
+			'wp-auth-check',
+			'wp-jquery-ui-dialog',
+			'wpfep-serial-notice-css',
+			'common',
+			'forms',
+			'admin-menu',
+			'dashboard',
+			'list-tables',
+			'edit',
+			'revisions',
```
Comment (zain-glowlogix) [inline inc/class-wpfep-roles-editor.php]: @mfaizanrasool please remove all unnecessary styles handles.

File: inc/class-wpfep-roles-editor.php (hunk 1 above)
Comment (mfaizanrasool) [reply inc/class-wpfep-roles-editor.php]: I have removed unnecessary scripts.

File: inc/class-wpfep-roles-editor.php (hunk 2 above)
Comment (mfaizanrasool) [reply inc/class-wpfep-roles-editor.php]: I have removed unnecessary styles.

**Pull request merged**